### DISCUSSION TITLE
record demo parameter?

### DISCUSSION THREAD

**Discussion by MrRocket** [sinshu/managed-doom#17] [General]
Hi, was just wondering since there's -playdemo and -timedemo if there's plans for a -record parameter?

Thanks

**Comment by sinshu** [comment]
No.

Since it's difficult to guarantee the demo playback compatibility with other ports, `-record` will not be supported.

**Reply by MrRocket** [reply]
How difficult do you think it would be to implement? as I might be able to, and then at least test it out and see.
I mean the parameter is in the original game.
I could see a few others being redundant also, but then there's also -turbo int which would make since to cap it at say 250, if you decided to use it. I'm mostly just curious as to what command line pars you might be adding at some point.

Thanks

**Reply by sinshu** [reply]
To implement `-record` itself should not be so hard.
The problem is that to prove the demo compatibility is hard.
To ensure the compatibility, I need to build a test framework something like [this](https://github.com/chocolate-doom/statcheck), but that requires a lot of work.
So I will not support `-record`.

Regarding other missing parameters, I don't have a concrete plan to add them.
But I believe that most parameters necessary to enjoy the game are available.
